src/sentry/static/sentry/app/utils/analytics.tsx

```diff
@@ -102,20 +118,25 @@ const CAN_MARK =
   typeof window.performance.getEntriesByName === 'function' &&
   typeof window.performance.clearMeasures === 'function';
 
-metric.mark = function metricMark(name) {
+metric.mark = function metricMark({name, data = {}}) {
```

Comment:
Changing the signature to take an object seems higher risk than adding in a `data` parameter, and doesn't seem to make types 'better'. Why take this approach over adding an optional data blob?

Reply:
Because `metric.measure` takes an object and it triggered my OCPD if `metric.mark` is different.

😬

Reply:
I think there's a nice simplicity to the previous `mark` api since there's only 1 required argument (name) and the the majority use case is just setting a mark w/o data.

Reply:
Fair point. I'm okay with changing it back if y'all are opinionated on this.

Reply:
I'll leave it up to you, I don't feel strongly about it -- in either case, do we want explicit types on the args?

Reply:
IMO having consistency on the args accepted by methods in `metric` is better, and the change affects only 6 other files which is fairly low-risk. I'll keep this PR as is.